index.html

```diff
@@ -1,12 +1,216 @@
+          creating a positive change with people from over 80 countries is
+          taking place in October, in Korea.
+        </p>
+        <div class="date-venue flex-column">
+          <h2 class="custom-gray-3 text-20-bold big-text-42">2023.10.19(THU) ~ 20(FRI)</h2>
+          <p class="custom-gray-4 text-13 big-text-27">
+            @ National Museum of Karea, Art Center Nabi and more
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- Program section-->
+    <section class="program-container">
+      <div class="program flex-column">
+        <div class="title flex-column">
+          <h4 class="custom-white text-20 flex-row">Main Program</h4>
+          <div class="title-underline"></div>
+        </div>
+        <div class="program-details">
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/lecture-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Lecture</h3>
+              <p class="custom-white text-10 big-text-12">
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/exhibition-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">CC Exhibition</h3>
+              <p class="custom-white text-10 big-text-12">
+                Appreciate various creations applying CC license of artists,
+                organized from Art Center Nabi
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/forum-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Forum</h3>
+              <p class="custom-white text-10 big-text-12">
+                Have the time to share your thoughts and opinions with experts
+                for each topic.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/workshop-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Workshop</h3>
+              <p class="custom-white text-10 big-text-12">
+                Try creating your own work using open source license rather than
+                just watching at it.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/ignite-icon.png" />
```

Comment:
[REQUIRED] Please include the alt attribute and a value in all image tags that are not just meant for styling or design. But add an empty string as the value if the image is meant for styling and beautification.